docs/kilt-baseline-replication.md

```diff
@@ -0,0 +1,136 @@
+# Replication guide KILT
+
+## Setup env
+
+```bash
+# Create a virtual env
+conda create -n kilt37 -y python=3.7 && conda activate kilt37
+
+# Get pyserini scripts and install package
+git clone https://github.com/castorini/pyserini.git
+pip install pyserini
+
+# Get KILT scripts and install package
+git clone https://github.com/facebookresearch/KILT.git
+cd KILT
+pip install -r requirements.txt
+pip install .
+
+# Get KILT queries and gold data
+mkdir data
+python scripts/donwload_all_kilt_data.py
+python scripts/get_triviaqa_input.py
+cd ..
+
+# Get the KILT knowledge source / wikipedia dump (34.76GiB)
+wget http://dl.fbaipublicfiles.com/KILT/kilt_knowledgesource.json
+
+# Get NLTK dependencies
+python -m nltk.downloader punkt
+python -m nltk.downloader stopwords
+```
+
+## Index the knowledge source
+
+Convert to passage or document level JSONL format indexable by Pyserini (takes 1-2hr if bigrams is enabled):
+
+```bash
+# We'll split it in multiple files to make this faster
+mkdir kilt_knowledge_split
```

Comment:
Unclear what directory you're in at the moment... this should probably be under `collections/` subdirectory so we can keep the directory structure clean?